### PR TITLE
Add status to ItemAction

### PR DESCRIPTION
**Pull request by mcellier**

### CODE REVIEW

File: src/itemAction/index.tsx [hunk 1]
```diff
@@ -1,44 +1,87 @@
...
+    if (status === ItemStatus.LOADING) {
+      rightAddon = <Loader
+        className={cc(prefix({ chevron: true }))}
+        size={24}
+        onDoneAnimationEnd={onDoneAnimationEnd}
+        inline
+      />
+    }
+    if (status === ItemStatus.CHECKED) {
+      rightAddon = (
```
Comment (Tuxosaurus) [inline src/itemAction/index.tsx]: myabe remove the `()` as is the case above

File: src/itemAction/index.tsx [hunk 2]
```diff
@@ -1,44 +1,87 @@
...
+      onBlur,
+      onFocus,
+      onMouseDown,
+      onDoneAnimationEnd,
+    } = this.props
+
+    let rightAddon
+
+    if (status === ItemStatus.LOADING) {
+      rightAddon = <Loader
```
Comment (Tuxosaurus) [inline src/itemAction/index.tsx]: maybe add `()` as is the case below

File: src/itemAction/index.unit.tsx [hunk 3]
```diff
@@ -50,4 +51,31 @@ describe('ItemAction', () => {
 
     expect(wrapper.prop('tag')).toEqual('li')
   })
+
+  describe('status', () => {
+    it('Should not render Loader whe status is not defined', () => {
```
Comment (Tuxosaurus) [inline src/itemAction/index.unit.tsx]: `whe` x2

File: src/itemAction/specifications/index.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+## Playground
+
+<!-- STORY -->
+
+## Specifications
+
+> Normal state
+
+These items always activate subactions and so modals (or sometimes native elements like a bottom sheet). We use them when the action is not part of the main flow it appears in (Contact the DRVR is a subaction of the booking flow, there’s a beginning and an end to it to then go back to the main flow)
```
Comment (Tuxosaurus) [inline src/itemAction/specifications/index.md]: `and so modals`?

**Comment by mcellier** [comment]
OMG, i didn't have the typescript check when doing this... Some fixes to make

**Pull request merged**